package/sdl2/sdl2.mk

```diff
@@ -72,27 +71,38 @@ SDL2_POST_INSTALL_STAGING_HOOKS += SDL2_FIX_SDL2_CONFIG_CMAKE
 # We must enable static build to get compilation successful.
 SDL2_CONF_OPTS += --enable-static
 
-# batocera
-# sdl2 set the rpi video output from the host name
+# batocera - disable hidapi
+SDL2_CONF_OPTS += --disable-hidapi
+
+# batocera - sdl2 set the rpi video output from the host name
 ifeq ($(BR2_PACKAGE_RPI_USERLAND),y)
 SDL2_CONF_OPTS += --host=arm-raspberry-linux-gnueabihf
 endif
 
-# batocera
-# Used in screen rotation (SDL and Retroarch)
+# batocera - Used in screen rotation (SDL and Retroarch)
 ifeq ($(BR2_PACKAGE_ROCKCHIP_RGA),y)
 SDL2_DEPENDENCIES += rockchip-rga
 endif
 
-# batocera
-# disable hidapi
-SDL2_CONF_OPTS += --disable-hidapi
-
-# batocera
-# pipewire
+# batocera - use Pipewire audio
 ifeq ($(BR2_PACKAGE_PIPEWIRE),y)
-SDL2_CONF_OPTS += --enable-pipewire
 SDL2_DEPENDENCIES += pipewire
+SDL2_CONF_OPTS += --enable-pipewire
+endif
+
+# batocera - enable/disable Wayland video driver
+ifeq ($(BR2_PACKAGE_WAYLAND),y)
+SDL2_DEPENDENCIES += wayland
+SDL2_CONF_OPTS += --enable-video-wayland
+else
+SDL2_CONF_OPTS += --disable-video-wayland
+endif
+
+# batocera - enable/disable Vulkan support
+ifeq ($(BR2_PACKAGE_BATOCERA_VULKAN),y)
```

Comment:
we should not use _BATOCERA_ values in buildroot.
instead, we should create a BR2_PACKAGE_SDL2_VULKAN variable
and set it if BR2_PACKAGE_BATOCERA_VULKAN is set